src/server/services/BruincastServices.js

```diff
@@ -32,9 +32,9 @@ class BruincastServices {
       course,
       {
         id: 69420,
-        label: '20S-MATH33B-1',
-        quarter: '20S',
-        title: 'Differential Equations',
+        label: '201-COMSCI32-1',
```

Comment:
@maxpluez Do you still need this fake data? Isn't the bccast end working already?

Reply:
I still need it for now. I can remove it once I finish my other ticket.